### ISSUE TITLE
Members dropdown list is massive

### ISSUE THREAD

**Issue by joseph-ortiz** [ggc-itec/ggc-itec.github.io#42]
As more members join, the dropdown list is growing big. This can lead to a bad user experience.

**Comment by tacksoo** [comment]
Yes, it currently doesn't look too good. I'll try two columns and see how it looks.

**Comment by tacksoo** [comment]
btw, any ideas why disqus isn't  working on discussions.html?

**Comment by joseph-ortiz** [comment]
@tacksoo looks like the bootstrap panel isn't loading the javascript correctly. There is a null reference exception for .appendChild() when the Immediate executing function in the disqus script tag.

In order to get it to load, the disqus_thread div needs to be moved outside the tabbable div in order for the javascript to execute. I still don't know the root cause.

Please see the open pull request for the proposed change.
